### DISCUSSION TITLE
Query Error : Error : self signed certificate in certificate chain in Postgres

### DISCUSSION THREAD

**Discussion by satriowicaksn** [Budibase/budibase#2758] [Help] [answered]
I have successfully deployed Budibase to Digital Ocean and it is now accessible. I'm having trouble adding Data -> External in Postgres. Error I encountered : [Query Error : Error : self signed certificate in certificate chain] . I don't think it's because of the configuration because I've tried to connect on other platforms with the same configuration and it worked.

Can anyone help or maybe have I had the problem with me ?

![image](https://user-images.githubusercontent.com/90515555/134944870-12e7a793-b41f-40d7-bfec-ca1cf7cd99cf.png)

**Comment by shogunpurple** [comment]
Hi @satrioasmaraku - we currently have an open issue for this here: https://github.com/Budibase/budibase/issues/1967

It's been scoped for the next few weeks so we will get it resolved ASAP.

**Reply by satriowicaksn** [reply]
hi @shogunpurple , I want to ask again whether the issue has been resolved and can be used? thank you

**Reply by shogunpurple** [reply]
As you can probably see, the issue is still open: https://github.com/Budibase/budibase/issues/1967

We will be working on this in our current iteration, and it will be solved in the coming weeks.

**Reply by satriowicaksn** [reply]
hi @shogunpurple is this issue has been solved ?

**Reply by shogunpurple** [reply]
Hi @satrioasmaraku yes, it's been released. You can add your CA cert to the postgres connector now and connect.

**Reply by satriowicaksn** [reply]
Hi @shogunpurple , i'm already add my CA certification , but i got new error here : 
![image](https://user-images.githubusercontent.com/90515555/143381331-97920db5-8014-4056-807a-5d6c0fcf8c13.png)

**Reply by shogunpurple** [reply]
Thanks @satrioasmaraku - would you be able to send the body of that response from the devtools in the browser, so that we can see what the error message is? Thanks

**Reply by satriowicaksn** [reply]
hi @shogunpurple , sure the response in dev tools is upstream request timeout 
![image](https://user-images.githubusercontent.com/90515555/143452125-59f44dc4-73d1-46e4-b792-3bcfa9da12c8.png)

**Reply by satriowicaksn** [reply]
![image](https://user-images.githubusercontent.com/90515555/143452234-0f096633-679f-43c0-80f1-78e5b885bf2d.png)

**Reply by satriowicaksn** [reply]
![image](https://user-images.githubusercontent.com/90515555/143452533-2cf05ee0-79f5-48d7-86d9-780467b2b7d4.png)

**Reply by shogunpurple** [reply]
Hi @satrioasmaraku - looks like the actual request is timing out completely to digitalocean. I will have to try this out with a digitalocean db in our cloud platform - it could be a networking issue

**Reply by satriowicaksn** [reply]
Thank you @shogunpurple , would you be able to tell me how about your connection with digitalocean db?

**Comment by satriowicaksn** [comment, marked as answer]
Solved after add CA certificates and whitelist Budibase IP in digital ocean
